test/unit/test_rtp.py

```diff
@@ -0,0 +1,247 @@
+        """
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='elv',
+                                   beam_num=7,
+                                   boundary=(0, 57))
+        plt.title("Simple Elevation no ground scatter, beam 7 Saskatoon plot")
+        plt.show()
+
+    def test_ground_scatter_range_time_plot(self):
+        """
+        plots a simple elevation rang-time plot with ground scatter
+        """
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='elv',
+                                   beam_num=7, ground_scatter=True)
+        plt.title("Elevation with Ground Scatter with beam 7")
+        plt.show()
+
+    def test_velocity_range_time_plot(self):
+        """
+        plots a velocity range-time plot for
+        beam 5 with the reversed jet colour mapping
+        """
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='v',
+                                   beam_num=5, color_map='jet_r')
+        plt.title("Velocity with reversed jet color map")
+        plt.show()
+
+    def test_velocity_error_range_time_plot(self):
+        """
+        plots velocity error range time plot which is not a pre-set
+        parameter also added extra settings for the color bar.
+        """
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='v_e',
+                                   beam_num=5, color_map='jet_r',
+                                   color_bar_label="velocity error $m/s$",
+                                   boundary=(-200, 200))
+        plt.title("Non-standard parameter: Velocity error\n"
+                  " with reversed jet color map")
+        plt.show()
+
+    def test_power_range_time_plot(self):
+        """
+        plots power range-time with ground scatter
+        """
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='p_l',
+                                   beam_num=7, ground_scatter=True)
+        plt.title("Elevation with Ground Scatter")
+        plt.show()
+
+    def test_multiplots_range_time_plt(self):
+        """
+        plots multiple range-time plots using subplots
+        """
+        plt.subplot(2, 1, 1)
+        plt.title("Subplots using plt method")
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='elv',
+                                   beam_num=7)
+        plt.subplot(2, 1, 2)
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='elv',
+                                   beam_num=15)
+        plt.show()
+
+    def test_axes_object_range_time_plot(self):
+        """
+        plots elevation range-time plot using axes object
+        """
+        fig, ax = plt.subplots()
+        pydarn.RTP.plot_range_time(self.fitacf_data, parameter='elv',
+                                   beam_num=7, ax=ax)
+        ax.set_xlabel("Date (UTC)")
+        ax.set_ylabel("Elevation $degrees$")
+        fig.suptitle("RTP plot using Axes object")
+        plt.show()
```

Comment:
Do we want to show things during tests?  I've run on some computers where I could only write out image files, not display them.

Reply:
Typically not, almost defeats the purpose of `python test`. However, the unit tests for plotting becomes very simplistic and testing becomes more visual.

Right if you `ssh` or don't have DISPLAY enabled on the machine then show() will not work. This was really to help me test it and ensure all the plots looked good. 

I think I will be making a discussion revolving around `python test` and I will include the plotting tests and what people would prefer as developers.